.prettierignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
sneak this in so that people with Prettier editor configs that _format-on-save_ don't accidentally change the code when working in Semiotic.